hr_expense_cancel/README.rst

```diff
@@ -0,0 +1,4 @@
+Expenses Cancel
+=============================
+
+Cancel an Expense paid or in Waiting payment 
```

Comment:
Complete the README with the full OCA's template: https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst